app/models/communication.rb

```diff
@@ -48,12 +48,11 @@ def self.create_with_twilio_details!(appointment:, twilio_sid:, twilio_msg_statu
   end
 
   def communication_result
-    case
-    when successful? then
+    if successful?
       COMMUNICATION_RESULTS[:successful]
-    when unsuccessful? then
+    elsif unsuccessful?
       COMMUNICATION_RESULTS[:unsuccessful]
-    when in_progress? then
+    elsif in_progress?
```

Comment:
This one bothers me, and I don't see this as anything really functional.  Will bring this one up for possible override elsewhere.